fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java

```diff
@@ -19,26 +19,31 @@
 import com.staros.util.LockCloseable;
 import com.starrocks.common.AlreadyExistsException;
 import com.starrocks.common.AnalysisException;
+import com.starrocks.common.Config;
 import com.starrocks.common.DdlException;
+import com.starrocks.credential.CloudConfigurationConstants;
 import com.starrocks.storagevolume.StorageVolume;
 
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 public class SharedDataStorageVolumeMgr extends StorageVolumeMgr {
-    @Override
-    public Long createStorageVolume(String name, String svType, List<String> locations, Map<String, String> params,
-                                    Optional<Boolean> enabled, String comment)
+    public static final String BUILTIN_STORAGE_VOLUME = "builtin_storage_volume";
+
+    public String createStorageVolume(String name, String svType, List<String> locations, Map<String, String> params,
+                                      Optional<Boolean> enabled, String comment)
             throws AlreadyExistsException, AnalysisException, DdlException {
         try (LockCloseable lock = new LockCloseable(rwLock.writeLock())) {
             if (exists(name)) {
                 throw new AlreadyExistsException(String.format("Storage volume '%s' already exists", name));
             }
-            StorageVolume sv = new StorageVolume(0, name, svType, locations, params, enabled.orElse(true), comment);
-            return Long.valueOf(GlobalStateMgr.getCurrentState().getStarOSAgent().addFileStore(sv.toFileStoreInfo()));
+            StorageVolume sv = new StorageVolume("", name, svType, locations, params, enabled.orElse(true), comment);
```

Comment:
it is a little weird to allow specify an empty volume id to create a Valid StorageVolume instance, how about add a static method
`public static FileStoreInfo StorageVolume::createFileStoreInfo(name, type, location, params, ...)`, inside this interface, it can use the private constructor with empty volume id.

This way, there is no way to leak an empty-id volume instance to the outside.